### PR TITLE
fix: show friendly messages for config assertions

### PR DESCRIPTION
**Pull request by sosukesuzuki**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -369,7 +381,7 @@ export function run(processArgv: readonly string[]): void {
       buildSoyOptions,
       async (argv) => {
         const config = loadConfig(argv);
-        assertNonNullable(config.soyOptions);
+        assertNonNullable(config, "soyOptions");
```
Comment (koba04) [inline src/cli.ts:384]: Is this suggestion correct?

```suggestion
        assertNonNullableWithConfig(config, "soyOptions");
```

Comment (sosukesuzuki) [reply src/cli.ts]: Yes! Thank you!

**Comment by koba04** [comment]
@sosukesuzuki 
Could you provide me how this PR changes the error message?

**Comment by sosukesuzuki** [comment]
Sure.

Run `duck build:soy` with the below `duck.config.js`.(Required properties are missing .)

```js
module.exports = {
  /* Required properties for building soy */
  // soyJarPath: "lib/closure-templates.jar",
  // soyFileRoots: ["src/soy"],
};
```

**before:**

We cannot know what is the required property.

```
TypeError: Should be string: undefined
    at Object.assertString (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/assert.js:6:15)
    at Object.handler (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/cli.js:264:18)
    at Object.runCommand (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:446:48)
    at Object.parseArgs [as _parseArgs] (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:2697:57)
    at Object.parse (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:2278:29)
    at Object.run (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/cli.js:320:10)
    at Object.<anonymous> (/development/duck-624-reproduce/node_modules/@cybozu/duck/bin/duck.js:3:23)
```

**after:**

We can know what is the required property.

```
TypeError: 'soyJarPath' is not string(undefined).: undefined
    at Object.assertString (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/assert.js:6:15)
    at Object.handler (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/cli.js:264:18)
    at Object.runCommand (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:446:48)
    at Object.parseArgs [as _parseArgs] (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:2697:57)
    at Object.parse (/development/duck-624-reproduce/node_modules/yargs/build/index.cjs:2278:29)
    at Object.run (/development/duck-624-reproduce/node_modules/@cybozu/duck/lib/cli.js:320:10)
    at Object.<anonymous> (/development/duck-624-reproduce/node_modules/@cybozu/duck/bin/duck.js:3:23)
```

**Review by koba04 — APPROVED** [review]
LGTM 👍 Thank you!

**Pull request merged**